azure-pipelines.yml

```diff
@@ -290,15 +301,12 @@ jobs:
     maxParallel: 4
 
   steps:
-  - task: CondaEnvironment@1
-    inputs:
-      createCustomEnvironment: true
-      environmentName: lkpy
-      packageSpecs: $(conda.deps)
-      updateConda: false
+  - bash: echo "##vso[task.prependpath]$CONDA/bin"
+    displayName: Add conda to PATH
+
+  - script: conda update -y conda
```

Comment:
I think you need to `sudo` this call because of where conda is installed on Hosted macOS

Reply:
Thanks! I'm going to just drop this update line; we'll live with a possibly outdated conda unless that starts causing problems.